tests/functions/module_custom.test.ts

```diff
@@ -129,4 +129,17 @@ describe("FORMAT.LARGE.NUMBER formula", () => {
 
     expect(evaluateCellText("A1", { A1: '=FORMAT.LARGE.NUMBER(100, "something")' })).toBe("#ERROR");
   });
+
+  test("Chaining FORMAT.LARGE.NUMBER does nothing", () => {
```

Comment:
maybe specify that it's specifically when we don't provide the second argument

Reply:
Yeah,  it's still broken if we chain the formula using different units ... Not sure if we can fix that. We could check that the suffix in the format looks like a large number suffix, but what if there's a currency that have the same look ?